pkg/aggregator/aggregator.go

```diff
@@ -272,7 +266,15 @@ func (agg *BufferedAggregator) AddAgentStartupTelemetry(agentVersion string) {
 		SampleRate: 1,
 		Timestamp:  0,
 	}
-	agg.eventIn <- event
+	if agg.hostname != "" {
+		// Send startup event only when we have a valid hostname
+		agg.eventIn <- metrics.Event{
+			Text:           fmt.Sprintf("Version %s", agentVersion),
+			SourceTypeName: "System",
+			Host:           agg.hostname,
+			EventType:      "Agent Startup",
+		}
+	}
 	agg.metricIn <- metric
```

Comment:
Super nit: let's put this under the creation of `metric`

Reply:
done 👍